README.md

```diff
@@ -187,10 +187,12 @@ This will create a `wit/world.wit` file describing the world that the
 component will target:
 
 ```wit
-default world component {
-  /// Say hello!
-  export hello-world: func() -> string
-}
+package component:example
```

Comment:
For someone figuring out what these names are and where they show up in the output, could you change either the `example` name here or the `example` name of the world below, so that they're different names?

Reply:
I'll update with a different package name.